content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -62,6 +62,12 @@ via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the
     $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system > $HOME/istio.yaml
     {{< /text >}}
 
+    > If you want to enable global mutual TLS, set `global.mtls.enabled` to `true`:
```

Comment:
I don't think you need the leading >

Reply:
Done.

Reply:
It may be helpful to explain (or at least point a link) why someone wants to set global.mtls.enabled=true.

I believe this isn't what we recommend to users unless they absolutely know what they are doing and have got their app running without mTLS.

Reply:
Sure. added the links to our concept doc.